pkg/controller/master/mcmsettings/fleetcontroller.go

```diff
@@ -0,0 +1,141 @@
+package mcmsettings
+
+import (
+	"encoding/base64"
+	"fmt"
+
+	mgmtv3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	"github.com/rancher/wrangler/pkg/relatedresource"
+	"github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	defaultFleetControllerConfigMapName      = "fleet-controller"
+	defaultFleetControllerConfigMapNamespace = "cattle-fleet-system"
```

Comment:
a better place for namespace definition is the `pkg/util/constants.go`

Reply:
good point.. i will move it there.